src-tauri/src/lib.rs

```diff
@@ -279,31 +279,24 @@ async fn do_install(
 pub async fn install_northstar(
     window: tauri::Window,
     game_path: &str,
-    northstar_package_name: Option<String>,
+    northstar_package_name: String,
+    version_number: Option<String>,
 ) -> Result<String, String> {
-    let northstar_package_name = match northstar_package_name {
-        Some(northstar_package_name) => {
-            if northstar_package_name.len() <= 1 {
-                "Northstar".to_string()
-            } else {
-                northstar_package_name
-            }
-        }
-        None => "Northstar".to_string(),
-    };
-
     let index = thermite::api::get_package_index().unwrap().to_vec();
     let nmod = index
         .iter()
         .find(|f| f.name.to_lowercase() == northstar_package_name.to_lowercase())
         .ok_or_else(|| panic!("Couldn't find Northstar on thunderstore???"))
         .unwrap();
 
+    // Use passed version or latest if no version was passed
+    let version = version_number.unwrap_or_else(|| nmod.latest.clone());
```

Comment:
this could be a borrow instead to avoid cloning and you borrow this anyway later

Reply:
So like this a769530b59c0a46813f1ac881dda628267438329 ? ^^"

Reply:
yes